models/candidate.rb

```diff
@@ -50,6 +50,7 @@ def as_json(options = nil)
         support_list: round_numbers(calculation(:support_list) || []),
         contributions_by_type: calculation(:contributions_by_type) || {},
         contributions_by_origin: calculation(:contributions_by_origin) || {},
+        small_contributions: calculation(:small_contributions) || {},
```

Comment:
This should be `calculation(:small_contributions).try(:to_f)` because it is a decimal when it exists. This will help the frontend be able to rely on its data type.

Also- in the automated build comment that was posted here to Github, it doesn't look like this calculation is defined for anyone? What do you make of that?